This uses a flag unrecognized by clang, also we do not need it.

--- a/arch/powerpc/boot/Makefile
+++ b/arch/powerpc/boot/Makefile
@@ -352,9 +352,6 @@ image-$(CONFIG_PPC_MICROWATT)		+= dtbImage.microwatt
 
 # For 32-bit powermacs, build the COFF and miboot images
 # as well as the ELF images.
-ifdef CONFIG_PPC32
-image-$(CONFIG_PPC_PMAC)	+= zImage.coff zImage.miboot
-endif
 
 # Allow extra targets to be added to the defconfig
 image-y	+= $(CONFIG_EXTRA_TARGETS)
